src/ktn_code.erl

```diff
@@ -26,15 +27,29 @@
       attrs => map(),
       node_attrs => map(),
       content => [tree_node()]}.
+-type beam_lib_beam() :: file:filename() | binary().
+% Should eventually become beam_lib:beam(), once that's exposed.
+-type erl_syntax_annotation_or_location() :: erl_anno:anno() | erl_anno:location().
+% Should eventually become erl_syntax:annotation_or_location(), once that's exposed.
+-type erl_parse_foo() ::
```

Comment:
I don't know where this `:foo` was coming from, but this is now apparent since we're forcing `dialyzer`'s `unknown`.